source/scripts/app.js

```diff
@@ -201,6 +211,8 @@ function bindExploreSearchBar() {
         let queryObj = {};
         queryObj.query = input.value; //Set query value to text
         queryObj.diet = "";
+        queryObj.cuisine = "";
+        queryObj.maxReadyTime = 240;
```

Comment:
We should make a constant for `240`, maybe something like `const DEFAULT_READY_TIME = 240;` at the top of the file